### PR TITLE
shellcheck: fix SC2006,SC2034,SC2242,SC2241,SC2239 warnings

### PR DESCRIPTION
**Pull request by v-sijliu**
Add SC2006,SC2034,SC2242,SC2241,SC2239 to flake8 and shellcheck

### CODE REVIEW

**Comment by LiliDeng** [comment]
@ader1990 Do you know why we still see SC2086 error even we didn't include it?

**Comment by LiliDeng** [comment]
Please fix remain errors. @v-sijliu

File: Testscripts/Linux/NET-Copy-Both-Ways.sh [hunk 1]
```diff
@@ -145,6 +145,7 @@ remote_home=$(ssh -i "$HOME"/.ssh/"$SSH_PRIVATE_KEY" -o StrictHostKeyChecking=no
     exit 0
     ")
 sts=$?
+LogMsg "$remote_home"
```
Comment (lizzha) [inline Testscripts/Linux/NET-Copy-Both-Ways.sh]: if the variable is not used, can we remove the "remote_home="  and run the command on the right side directly?

File: Testscripts/Linux/NET-Test-Network.sh [hunk 2]
```diff
@@ -189,6 +189,7 @@ if [ "${MAC:-UNDEFINED}" != "UNDEFINED" ]; then
     else
         LogMsg "MAC Address $MAC was found on the VM"   
     fi
+    LogMsg "$__sys_interface"
```
Comment (lizzha) [inline Testscripts/Linux/NET-Test-Network.sh]: ditto

File: Testscripts/Linux/STOR-Lis-Disk.sh [hunk 3]
```diff
@@ -82,6 +82,7 @@ LogMsg "constants disk count = $disk_count"
 #
 sd_count=0
 for drive in /dev/sd*[^0-9]; do
+    LogMsg "$drive"
     sd_count=$((sd_count + 1))
 done
```
Comment (lizzha) [inline Testscripts/Linux/STOR-Lis-Disk.sh]: can use command "ls /dev/sd*[^0-9] | wc -l" to count the number. Thus the variable "drive" is not needed

File: Testscripts/Linux/TestRDMA_MultiVM.sh [hunk 4]
```diff
@@ -90,6 +89,7 @@ for vm in $master $slaves_array; do
 	LogMsg "Checking $ib_nic status in $vm"
 	temp=$(ssh root@${vm} "ifconfig $ib_nic | grep 'inet '")
 	ib_nic_status=$?
+	LogMsg "$temp"
```
Comment (lizzha) [inline Testscripts/Linux/TestRDMA_MultiVM.sh]: Run it directly? ssh root@${vm} "ifconfig $ib_nic | grep 'inet '"

File: Testscripts/Linux/customKernelInstall.sh [hunk 5]
```diff
@@ -63,6 +63,7 @@ CheckInstallLockUbuntu()
     else
         LogMsg "No lock on dpkg present."
     fi
+    LogMsg "$dpkgPID"
```
Comment (lizzha) [inline Testscripts/Linux/customKernelInstall.sh]: ditto

File: Testscripts/Linux/perf_fio_nfs.sh [hunk 6]
```diff
@@ -165,6 +163,7 @@ if [ $? -eq 0 ]; then
 	else
 		mdVolume="/dev/md0"
 	fi
+	LogMsg "$mdVolume"
```
Comment (lizzha) [inline Testscripts/Linux/perf_fio_nfs.sh]: please check is this variable used anywhere?

File: Testscripts/Linux/utils.sh [hunk 7]
```diff
@@ -337,7 +337,7 @@ GetDistro()
 			return 1
 		;;
 	esac
-
+	echo "$OS_FAMILY"
```
Comment (lizzha) [inline Testscripts/Linux/utils.sh]: Print more messages, "OS family: $OS_FAMILY"

File: Testscripts/Linux/utils.sh [hunk 8]
```diff
@@ -2089,6 +2090,7 @@ GetGuestGeneration()
     else
         os_GENERATION=1
     fi
+	echo "$os_GENERATION"
```
Comment (lizzha) [inline Testscripts/Linux/utils.sh]: Generation: $os_GENERATION

File: Testscripts/Linux/utils.sh [hunk 9]
```diff
@@ -3284,6 +3286,7 @@ function test_rsync_files() {
 function change_mtu_increment() {
     test_iface=$1
     ignore_iface=$2
+    echo "$ignore_iface"
```
Comment (lizzha) [inline Testscripts/Linux/utils.sh]: it seems a bug in this function, the variable used later is $iface_ignore, please fix it.

**Comment by ader1990** [comment]
@v-sijliu please add in the azure-pipelines.yml the shellcheck stage and rules that you are fixing. you can get the stage from the https://github.com/LIS/LISAv2/pull/447

File: Testscripts/Linux/BVT-NET-IFUP-IFDOWN.sh [hunk 10]
```diff
@@ -61,7 +61,7 @@ ReloadNetvsc()
 # Source constants file and initialize most common variables
```
Comment (ader1990) [inline Testscripts/Linux/BVT-NET-IFUP-IFDOWN.sh]: @v-sijliu please add in the azure-pipelines.yml the shellcheck stage and rules that you are fixing. you can get the stage from the #447

File: Testscripts/Linux/NET-Max-NICs.sh [hunk 11]
```diff
@@ -64,9 +64,6 @@ if [ "${TEST_TYPE:-UNDEFINED}" = "UNDEFINED" ]; then
     LogMsg "Error : Parameter TEST_TYPE was not found"
     SetTestStateAborted
     exit 0
-else
-    IFS=',' read -a TYPE <<< "$TEST_TYPE"
-fi
```
Comment (LiliDeng) [inline Testscripts/Linux/NET-Max-NICs.sh]: keep fi

File: Testscripts/Linux/BVT-NET-IFUP-IFDOWN.sh (hunk 10 above)
Comment (LiliDeng) [reply Testscripts/Linux/BVT-NET-IFUP-IFDOWN.sh]: @ader1990 already change azure-pipelines.yml  in this PR

**Comment by LiliDeng** [comment]
For nested ones, pick up one case is enough, because only update the same command

  | Azure | Result | HyperV
-- | -- | -- | --
//Testscripts/Linux/BVT-NET-IFUP-IFDOWN.sh   (Edit) | BVT-NET-IFUP-IFDOWN | PASS | BVT-NET-IFUP-IFDOWN
//Testscripts/Linux/CollectLogFile.sh   (Edit) |   |   |  
//Testscripts/Linux/ConfigureSRIOV.sh   (Edit) | VERIFY-ENABLE-DISABLE-SRIOV-ON-EXISTING-VM | PASS |  
//Testscripts/Linux/ConfigureSRIOV-workaround.sh   (Edit) |   |   |  
//Testscripts/Linux/CORE-LogCollector.sh   (Edit) |   |   |  
//Testscripts/Linux/CreateRaid.sh   (Edit) | PERF-STORAGE-OVER-NFS-Synthetic-TCP-4K | PASS |  
//Testscripts/Linux/customKernelInstall.sh   (Edit) | -CustomKernel "http://linuxpipelineshare.blob.core.windows.net/linuxpackages/linuxnext-kernel-latest.deb | PASS |  
//Testscripts/Linux/customLISInstall.sh   (Edit) |   |   |  
//Testscripts/Linux/fio_jason_parser.sh (Edit) （7）（10）（21）（23） | PERF-STORAGE-4K-IO | PASS | HYPERV-WINDOWS-NESTED-HYPERV-STORAGE-SINGLE-DISK
//Testscripts/Linux/InstallCUDADrivers.sh   (Edit) | VERIFY-NVIDIA-CUDA-DRIVER-FOR-NC6-VM   canonical   ubuntuserver 16.04-lts Latest | PASS |  
//Testscripts/Linux/LIS-Mount-FloppyDisk.sh   (Edit) |   |   | LIS-MOUNT-FLOPPYDISK
//Testscripts/Linux/nested_hyperv_ntttcp_different_l1_nat.sh   (Edit) |   |   | NESTED-HYPERV-NTTTCP-DIFFERENT-L1-NAT
//Testscripts/Linux/nested_hyperv_ntttcp_different_l1_public_bridge.sh   (Edit) |   |   | NESTED-HYPERV-NTTTCP-DIFFERENT-L1-PUBLIC-BRIDGE
//Testscripts/Linux/nested_kvm_lagscope_different_l1_public_bridge.sh   (Edit) |   |   | NESTED-KVM-LAGSCOPE-DIFFERENT-L1-PUBLIC-BRIDGE
//Testscripts/Linux/nested_kvm_netperf_pps.sh   (Edit) |   |   | NESTED-KVM-NETPERF-PPS
//Testscripts/Linux/nested_kvm_netperf_pps_nat.sh   (Edit) | AZURE-NESTED-KVM-NETPERF-PPS | Pass |  
//Testscripts/Linux/nested_kvm_ntttcp_different_l1_nat.sh   (Edit) | AZURE-NESTED-KVM-NTTTCP-DIFFERENT-L1-NAT |   | NESTED-KVM-NTTTCP-DIFFERENT-L1-NAT
//Testscripts/Linux/nested_kvm_ntttcp_different_l1_public_bridge.sh   (Edit) |   |   | NESTED-KVM-NTTTCP-DIFFERENT-L1-PUBLIC-BRIDGE
//Testscripts/Linux/nested_kvm_ntttcp_private_bridge.sh   (Edit) | NESTED-KVM-NTTTCP-PRIVATE-BRIDGE | Pass |  
//Testscripts/Linux/nested_kvm_perf_fio.sh   (Edit) |   |   |  
//Testscripts/Linux/nested_kvm_storage_perf.sh   (Edit) | AZURE-NESTED-KVM-STORAGE-SINGLE-DISK |   | HYPERV-NESTED-KVM-STORAGE-SINGLE-DISK
//Testscripts/Linux/nested_vm_utils.sh (Edit)    （13） |   |   |  
//Testscripts/Linux/NET-Configure-Vxlan.sh   (Edit) |   |   |  
//Testscripts/Linux/NET-Copy-Both-Ways.sh   (Edit) |   |   | NET-COPY-BOTH-WAYS
//Testscripts/Linux/NET-Max-NICs.sh   (Edit) |   |   | NET-MAX-SYNTHETIC-NICS   add fi
//Testscripts/Linux/NET-Netperf-Server.sh   (Edit) |   |   | ETHTOOL-CHECK-STATISTICS
//Testscripts/Linux/NET-Test-Legacy.sh   (Edit) |   |   | NET-LEGACY
//Testscripts/Linux/NET-Test-Network.sh   (Edit) |   |   | NET-EXTERNAL
//Testscripts/Linux/perf_fio.sh   (Edit) | PERF-STORAGE-4K-IO | PASS | PERF-STORAGE-4K-IO
//Testscripts/Linux/perf_fio_nfs.sh (Edit) | PERF-STORAGE-OVER-NFS-Synthetic-TCP-4K |   |  
//Testscripts/Linux/STOR_VHDXResize_PartitionDisk.sh   (Edit) |   |   | VHDX-RESIZE-TO-BIGGER512
//Testscripts/Linux/STOR_VHDXResize_PartitionDiskOver2TB.sh   (Edit) |   |   | VHDX-RESIZE-OVER2TB-SCSI
//Testscripts/Linux/STOR_VSS_Disk_Stress.sh   (Edit) |   |   | STOR-VSS-BACKUP-RESTORE-DISKSTRESS
//Testscripts/Linux/STOR_VSS_ISCSI_PartitionDisks.sh   (Edit) |   |   | PRODUCTION-CHECKPOINT-iSCSI
//Testscripts/Linux/STOR_VSS_Set_VSS_Daemon.sh   (Edit) |   |   | STOR-VSS-BACKUP-CHANGE-HYPERVVSSD
//Testscripts/Linux/STOR_VSS_StopNetwork.sh   (Edit) |   |   | PRODUCTION-CHECKPOINT-ISO-NONETWORK
//Testscripts/Linux/STOR-Lis-Disk.sh   (Edit) |   |   | STORAGE-VHDX-IDE-FIXED
//Testscripts/Linux/Sysbench.sh   (Edit) |   |   | STRESSTEST-SYSBENCH
//Testscripts/Linux/TestRDMA_MultiVM.sh   (Edit) | VERIFY-INFINIBAND-INTEL-MPI-2VM-STANDARD-A8 | Aborted |  
//Testscripts/Linux/verify-ifdown-ifup-eth0-rmmod-modprobe-hv-netvsc.sh   (Edit) | VERIFY-IFUP-IFDOWN-RELOAD-HV_NETVSC-Synthetic   canonical   ubuntuserver 16.04-lts Latest | Pass |  


  | Azure | Result | HyperV | Result
-- | -- | -- | -- | --
//Testscripts/Linux/BVT-NET-IFUP-IFDOWN.sh   (Edit) | BVT-NET-IFUP-IFDOWN | PASS | BVT-NET-IFUP-IFDOWN |  
//Testscripts/Linux/CollectLogFile.sh   (Edit) |   |   |   |  
//Testscripts/Linux/ConfigureSRIOV.sh   (Edit) | VERIFY-ENABLE-DISABLE-SRIOV-ON-EXISTING-VM | PASS |   |  
//Testscripts/Linux/ConfigureSRIOV-workaround.sh   (Edit) |   |   |   |  
//Testscripts/Linux/CORE-LogCollector.sh   (Edit) |   |   |   |  
//Testscripts/Linux/CreateRaid.sh   (Edit) | PERF-STORAGE-OVER-NFS-Synthetic-TCP-4K | PASS |   |  
//Testscripts/Linux/customKernelInstall.sh   (Edit) | -CustomKernel "http://linuxpipelineshare.blob.core.windows.net/linuxpackages/linuxnext-kernel-latest.deb | PASS |   |  
//Testscripts/Linux/customLISInstall.sh   (Edit) |   |   |   |  
//Testscripts/Linux/fio_jason_parser.sh (Edit) （7）（10）（21）（23） | PERF-STORAGE-4K-IO | PASS | HYPERV-WINDOWS-NESTED-HYPERV-STORAGE-SINGLE-DISK |  
//Testscripts/Linux/InstallCUDADrivers.sh   (Edit) | VERIFY-NVIDIA-CUDA-DRIVER-FOR-NC6-VM   canonical   ubuntuserver 16.04-lts Latest | PASS |   |  
//Testscripts/Linux/LIS-Mount-FloppyDisk.sh   (Edit) |   |   | LIS-MOUNT-FLOPPYDISK | PASS
//Testscripts/Linux/nested_hyperv_ntttcp_different_l1_nat.sh   (Edit) |   |   | NESTED-HYPERV-NTTTCP-DIFFERENT-L1-NAT |  
//Testscripts/Linux/nested_hyperv_ntttcp_different_l1_public_bridge.sh   (Edit) |   |   | NESTED-HYPERV-NTTTCP-DIFFERENT-L1-PUBLIC-BRIDGE |  
//Testscripts/Linux/nested_kvm_lagscope_different_l1_public_bridge.sh   (Edit) |   |   | NESTED-KVM-LAGSCOPE-DIFFERENT-L1-PUBLIC-BRIDGE |  
//Testscripts/Linux/nested_kvm_netperf_pps.sh   (Edit) |   |   | NESTED-KVM-NETPERF-PPS |  
//Testscripts/Linux/nested_kvm_netperf_pps_nat.sh   (Edit) | AZURE-NESTED-KVM-NETPERF-PPS | Pass |   |  
//Testscripts/Linux/nested_kvm_ntttcp_different_l1_nat.sh   (Edit) | AZURE-NESTED-KVM-NTTTCP-DIFFERENT-L1-NAT |   | NESTED-KVM-NTTTCP-DIFFERENT-L1-NAT |  
//Testscripts/Linux/nested_kvm_ntttcp_different_l1_public_bridge.sh   (Edit) |   |   | NESTED-KVM-NTTTCP-DIFFERENT-L1-PUBLIC-BRIDGE |  
//Testscripts/Linux/nested_kvm_ntttcp_private_bridge.sh   (Edit) | NESTED-KVM-NTTTCP-PRIVATE-BRIDGE | Pass |   |  
//Testscripts/Linux/nested_kvm_perf_fio.sh   (Edit) |   |   |   |  
//Testscripts/Linux/nested_kvm_storage_perf.sh   (Edit) | AZURE-NESTED-KVM-STORAGE-SINGLE-DISK |   | HYPERV-NESTED-KVM-STORAGE-SINGLE-DISK |  
//Testscripts/Linux/nested_vm_utils.sh (Edit)    （13） |   |   |   |  
//Testscripts/Linux/NET-Configure-Vxlan.sh   (Edit) |   |   |   |  
//Testscripts/Linux/NET-Copy-Both-Ways.sh   (Edit) |   |   | NET-COPY-BOTH-WAYS | PASS
//Testscripts/Linux/NET-Max-NICs.sh   (Edit) |   |   | NET-MAX-SYNTHETIC-NICS   add fi | PASS
//Testscripts/Linux/NET-Netperf-Server.sh   (Edit) |   |   | ETHTOOL-CHECK-STATISTICS | Fail   same with master
//Testscripts/Linux/NET-Test-Legacy.sh   (Edit) |   |   | NET-LEGACY | PASS
//Testscripts/Linux/NET-Test-Network.sh   (Edit) |   |   | NET-EXTERNAL | PASS
//Testscripts/Linux/perf_fio.sh   (Edit) | PERF-STORAGE-4K-IO | PASS | PERF-STORAGE-4K-IO |  
//Testscripts/Linux/perf_fio_nfs.sh (Edit) | PERF-STORAGE-OVER-NFS-Synthetic-TCP-4K |   |   |  
//Testscripts/Linux/STOR_VHDXResize_PartitionDisk.sh   (Edit) |   |   | VHDX-RESIZE-TO-BIGGER512 | PASS
//Testscripts/Linux/STOR_VHDXResize_PartitionDiskOver2TB.sh   (Edit) |   |   | VHDX-RESIZE-OVER2TB-SCSI | PASS
//Testscripts/Linux/STOR_VSS_Disk_Stress.sh   (Edit) |   |   | STOR-VSS-BACKUP-RESTORE-DISKSTRESS |  
//Testscripts/Linux/STOR_VSS_ISCSI_PartitionDisks.sh   (Edit) |   |   | PRODUCTION-CHECKPOINT-iSCSI |  
//Testscripts/Linux/STOR_VSS_Set_VSS_Daemon.sh   (Edit) |   |   | STOR-VSS-BACKUP-CHANGE-HYPERVVSSD | PASS
//Testscripts/Linux/STOR_VSS_StopNetwork.sh   (Edit) |   |   | PRODUCTION-CHECKPOINT-ISO-NONETWORK | PASS
//Testscripts/Linux/STOR-Lis-Disk.sh   (Edit) |   |   | STORAGE-VHDX-IDE-FIXED | PASS
//Testscripts/Linux/Sysbench.sh   (Edit) |   |   | STRESSTEST-SYSBENCH | PASS
//Testscripts/Linux/TestRDMA_MultiVM.sh   (Edit) | VERIFY-INFINIBAND-INTEL-MPI-2VM-STANDARD-A8 | Aborted |   |  
//Testscripts/Linux/verify-ifdown-ifup-eth0-rmmod-modprobe-hv-netvsc.sh   (Edit) | VERIFY-IFUP-IFDOWN-RELOAD-HV_NETVSC-Synthetic   canonical   ubuntuserver 16.04-lts Latest | Pass |   |

**Comment by LiliDeng** [comment]
For nested ones, pick up one case is enough, because only update the same command


  | Azure | Result | HyperV
-- | -- | -- | --
//Testscripts/Linux/BVT-NET-IFUP-IFDOWN.sh   (Edit) | BVT-NET-IFUP-IFDOWN | PASS | BVT-NET-IFUP-IFDOWN
//Testscripts/Linux/CollectLogFile.sh   (Edit) |   |   |  
//Testscripts/Linux/ConfigureSRIOV.sh   (Edit) | VERIFY-ENABLE-DISABLE-SRIOV-ON-EXISTING-VM | PASS |  
//Testscripts/Linux/ConfigureSRIOV-workaround.sh   (Edit) |   |   |  
//Testscripts/Linux/CORE-LogCollector.sh   (Edit) |   |   |  
//Testscripts/Linux/CreateRaid.sh   (Edit) | PERF-STORAGE-OVER-NFS-Synthetic-TCP-4K | PASS |  
//Testscripts/Linux/customKernelInstall.sh   (Edit) | -CustomKernel "http://linuxpipelineshare.blob.core.windows.net/linuxpackages/linuxnext-kernel-latest.deb | PASS |  
//Testscripts/Linux/customLISInstall.sh   (Edit) |   |   |  
//Testscripts/Linux/fio_jason_parser.sh (Edit) （7）（10）（21）（23） | PERF-STORAGE-4K-IO | PASS | HYPERV-WINDOWS-NESTED-HYPERV-STORAGE-SINGLE-DISK
//Testscripts/Linux/InstallCUDADrivers.sh   (Edit) | VERIFY-NVIDIA-CUDA-DRIVER-FOR-NC6-VM   canonical   ubuntuserver 16.04-lts Latest | PASS |  
//Testscripts/Linux/LIS-Mount-FloppyDisk.sh   (Edit) |   |   | LIS-MOUNT-FLOPPYDISK
//Testscripts/Linux/nested_hyperv_ntttcp_different_l1_nat.sh   (Edit) |   |   | NESTED-HYPERV-NTTTCP-DIFFERENT-L1-NAT
//Testscripts/Linux/nested_hyperv_ntttcp_different_l1_public_bridge.sh   (Edit) |   |   | NESTED-HYPERV-NTTTCP-DIFFERENT-L1-PUBLIC-BRIDGE
//Testscripts/Linux/nested_kvm_lagscope_different_l1_public_bridge.sh   (Edit) |   |   | NESTED-KVM-LAGSCOPE-DIFFERENT-L1-PUBLIC-BRIDGE
//Testscripts/Linux/nested_kvm_netperf_pps.sh   (Edit) |   |   | NESTED-KVM-NETPERF-PPS
//Testscripts/Linux/nested_kvm_netperf_pps_nat.sh   (Edit) | AZURE-NESTED-KVM-NETPERF-PPS | Pass |  
//Testscripts/Linux/nested_kvm_ntttcp_different_l1_nat.sh   (Edit) | AZURE-NESTED-KVM-NTTTCP-DIFFERENT-L1-NAT |   | NESTED-KVM-NTTTCP-DIFFERENT-L1-NAT
//Testscripts/Linux/nested_kvm_ntttcp_different_l1_public_bridge.sh   (Edit) |   |   | NESTED-KVM-NTTTCP-DIFFERENT-L1-PUBLIC-BRIDGE
//Testscripts/Linux/nested_kvm_ntttcp_private_bridge.sh   (Edit) | NESTED-KVM-NTTTCP-PRIVATE-BRIDGE | Pass |  
//Testscripts/Linux/nested_kvm_perf_fio.sh   (Edit) |   |   |  
//Testscripts/Linux/nested_kvm_storage_perf.sh   (Edit) | AZURE-NESTED-KVM-STORAGE-SINGLE-DISK |   | HYPERV-NESTED-KVM-STORAGE-SINGLE-DISK
//Testscripts/Linux/nested_vm_utils.sh (Edit)    （13） |   |   |  
//Testscripts/Linux/NET-Configure-Vxlan.sh   (Edit) |   |   |  
//Testscripts/Linux/NET-Copy-Both-Ways.sh   (Edit) |   |   | NET-COPY-BOTH-WAYS
//Testscripts/Linux/NET-Max-NICs.sh   (Edit) |   |   | NET-MAX-SYNTHETIC-NICS   add fi
//Testscripts/Linux/NET-Netperf-Server.sh   (Edit) |   |   | ETHTOOL-CHECK-STATISTICS
//Testscripts/Linux/NET-Test-Legacy.sh   (Edit) |   |   | NET-LEGACY
//Testscripts/Linux/NET-Test-Network.sh   (Edit) |   |   | NET-EXTERNAL
//Testscripts/Linux/perf_fio.sh   (Edit) | PERF-STORAGE-4K-IO | PASS | PERF-STORAGE-4K-IO
//Testscripts/Linux/perf_fio_nfs.sh (Edit) | PERF-STORAGE-OVER-NFS-Synthetic-TCP-4K |   |  
//Testscripts/Linux/STOR_VHDXResize_PartitionDisk.sh   (Edit) |   |   | VHDX-RESIZE-TO-BIGGER512
//Testscripts/Linux/STOR_VHDXResize_PartitionDiskOver2TB.sh   (Edit) |   |   | VHDX-RESIZE-OVER2TB-SCSI
//Testscripts/Linux/STOR_VSS_Disk_Stress.sh   (Edit) |   |   | STOR-VSS-BACKUP-RESTORE-DISKSTRESS
//Testscripts/Linux/STOR_VSS_ISCSI_PartitionDisks.sh   (Edit) |   |   | PRODUCTION-CHECKPOINT-iSCSI
//Testscripts/Linux/STOR_VSS_Set_VSS_Daemon.sh   (Edit) |   |   | STOR-VSS-BACKUP-CHANGE-HYPERVVSSD
//Testscripts/Linux/STOR_VSS_StopNetwork.sh   (Edit) |   |   | PRODUCTION-CHECKPOINT-ISO-NONETWORK
//Testscripts/Linux/STOR-Lis-Disk.sh   (Edit) |   |   | STORAGE-VHDX-IDE-FIXED
//Testscripts/Linux/Sysbench.sh   (Edit) |   |   | STRESSTEST-SYSBENCH
//Testscripts/Linux/TestRDMA_MultiVM.sh   (Edit) | VERIFY-INFINIBAND-INTEL-MPI-2VM-STANDARD-A8 | Aborted |  
//Testscripts/Linux/verify-ifdown-ifup-eth0-rmmod-modprobe-hv-netvsc.sh   (Edit) | VERIFY-IFUP-IFDOWN-RELOAD-HV_NETVSC-Synthetic   canonical   ubuntuserver 16.04-lts Latest | Pass |

**Comment by LiliDeng** [comment]
@lizzha Done for testing.

BVT-NET-IFUP-IFDOWN.sh
BVT-NET-IFUP-IFDOWN PASS

ConfigureSRIOV.sh
VERIFY-ENABLE-DISABLE-SRIOV-ON-EXISTING-VM PASS

CreateRaid.sh
PERF-STORAGE-OVER-NFS-Synthetic-TCP-4K PASS

customKernelInstall.sh
-CustomKernel "http://linuxpipelineshare.blob.core.windows.net/linuxpackages/linuxnext-kernel-latest.deb PASS

customLISInstall.sh
-CustomLIS lisnext PASS

fio_jason_parser.sh
PERF-STORAGE-4K-IO PASS

InstallCUDADrivers.sh
VERIFY-NVIDIA-CUDA-DRIVER-FOR-NC6-VM PASS

LIS-Mount-FloppyDisk.sh
LIS-MOUNT-FLOPPYDISK PASS

nested_kvm_netperf_pps.sh
NESTED-KVM-NETPERF-PPS Pass

nested_kvm_netperf_pps_nat.sh
AZURE-NESTED-KVM-NETPERF-PPS Pass

nested_kvm_ntttcp_private_bridge.sh
NESTED-KVM-NTTTCP-PRIVATE-BRIDGE Pass

NET-Configure-Vxlan.sh
NET-Copy-Both-Ways.sh
NET-COPY-BOTH-WAYS Pass

NET-Max-NICs.sh 
NET-MAX-SYNTHETIC-NICS Pass

NET-Netperf-Server.sh
ETHTOOL-CHECK-STATISTICS Fail, same with master

NET-Test-Legacy.sh
NET-LEGACY Pass

NET-Test-Network.sh
NET-EXTERNAL Pass

perf_fio.sh
PERF-STORAGE-4K-IO PASS

perf_fio_nfs.sh
PERF-STORAGE-OVER-NFS-Synthetic-TCP-4K PASS

STOR_VHDXResize_PartitionDisk.sh
VHDX-RESIZE-TO-BIGGER512 PASS

STOR_VHDXResize_PartitionDiskOver2TB.sh
VHDX-RESIZE-OVER2TB-SCSI PASS

STOR_VSS_Disk_Stress.sh
STOR-VSS-BACKUP-RESTORE-DISKSTRESS Fail, same with master

STOR_VSS_ISCSI_PartitionDisks.sh
PRODUCTION-CHECKPOINT-iSCSI Fail, same with master
 
STOR_VSS_Set_VSS_Daemon.sh
STOR-VSS-BACKUP-CHANGE-HYPERVVSSD PASS

STOR_VSS_StopNetwork.sh
PRODUCTION-CHECKPOINT-ISO-NONETWORK PASS

STOR-Lis-Disk.sh
STORAGE-VHDX-IDE-FIXED PASS

Sysbench.sh
STRESSTEST-SYSBENCH PASS

TestRDMA_MultiVM.sh
VERIFY-INFINIBAND-INTEL-MPI-2VM-STANDARD-A8
Same result with master, download log timeout

verify-ifdown-ifup-eth0-rmmod-modprobe-hv-netvsc.sh
VERIFY-IFUP-IFDOWN-RELOAD-HV_NETVSC-Synthetic Pass

File: Testscripts/Linux/utils.sh [hunk 12]
```diff
@@ -686,6 +686,7 @@ NetmaskToCidr()
 
 	declare -i netbits=0
 	oldifs="$IFS"
+	echo "$oldifs"
```
Comment (lizzha) [inline Testscripts/Linux/utils.sh]: this variable is not used, can be removed

**Comment by juhlee-microsoft** [comment]
@LiliDeng are we still working on 4 errors? @adriansuhov fixed the log downloading timeout in setupRDMA branch before. Once it is merging to master, below problem will be resolved. 


_**TestRDMA_MultiVM.sh
VERIFY-INFINIBAND-INTEL-MPI-2VM-STANDARD-A8
Same result with master, download log timeout**_

Need to clarify what we are going to do with remaining 4 failed cases. @lizzha already signed off.

**Comment by LiliDeng** [comment]
@juhlee-microsoft You mean the conflict? We will fix it.

**Comment by LiliDeng** [comment]
@juhlee-microsoft We are sure that those failures not cause by our change.

**Comment by lizzha** [comment]
@juhlee-microsoft for the failed cases, if they're test issues, Lili's team will fix them. They're cleaning up test issues on Azure case by case now.
When do you plan to merge the setupRDMA branch to master?

**Comment by juhlee-microsoft** [comment]
Plan it around beginning of Feb.

________________________________

**Comment by LiliDeng** [comment]
Pick up 2 cases to run against latest code, passed, this PR is ready for merged @lizzha Thanks!

**Pull request merged**